updateagent/agent.py

```diff
@@ -1,6 +1,6 @@
 import re
 import string
-from types import StringType
+#from types import StringType
```

Comment:
Let's leave the changes in updateagent out of this PR, and put them in another PR.